app/views/projects/show.html.erb

```diff
@@ -264,6 +264,68 @@
   </div>
 </div>
 
+<div class="row">
+  <div class="page-header">
+    <h2>Deprecated Resources</h2>
+  </div>
+  <!-- :pdrive_folders, :dropbox_folders, :pivotal_tracker_ids, :trello_ids -->
+  <div class="col-lg-6">
+    <p><strong>P Drive Folders</strong></p>
+    <p>
+     <% if !@project.pdrive_folders.blank? %>
+        <ul>
+          <% @project.pdrive_folders.each do |folder| %>
+            <li><%= folder %></li>
+          <% end %>
+        </ul>
+      <% else %>
+        This project has no P Drive Folders
+      <% end %>
+    </p>
```

Comment:
To save cluttering, for deprecated resources, could we maybe have the if around the whole section and just not show that header if it isn't present? E.g. if no P Drive folders we don't show the p drive section at all, rather than showing 'This project has no P Drive Folders'.

What do you think?

Reply:
Ah I see, yeah I think the else statement is just for presentation purposes - i.e. not having a blank space. I can leave it out, no problem

Reply:
Ah sorry what I mean is that the header `<p><strong>P Drive Folders</strong></p>` would also be moved inside the if.